src/actions/loginAction.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { LOGIN_USER_SUCCESS, LOGIN_USER_FAILED } from '../reducers/auth/login/actionTypes';
+import history from '../utils/history';
+
+const apiUrl = 'https://ah-backend-kronos-staging.herokuapp.com/api/users/login/';
+
+
+const loginUserAction = userData => dispatch => axios.post(apiUrl, userData)
+  .then((response) => {
+    dispatch({
+      type: LOGIN_USER_SUCCESS,
+      payload: response.data,
+    });
+    toast.dismiss();
+    toast.success(`Welcome ${response.data.user.username}. Login Successful`, {
+      containerId: 'A',
+      position: toast.POSITION.TOP_RIGHT,
+      autoClose: false,
+      hideProgressBar: false,
+    });
+    sessionStorage.setItem('userToken', response.data.user.auth_token);
+    sessionStorage.setItem('username', response.data.user.username);
+    sessionStorage.setItem('email', response.data.user.email);
+    history.push('/successpage');
```

Comment:
You have to set the logged_in attribute in the session because it is being used in the application to see that a user is logged_in. 
Email attribute in the session is not necessary since after logging in we go to the article pages we no longer have a success page

Reply:
Resolved.